source/extensions/quic_listeners/quiche/platform/quic_logging_impl.cc

```diff
@@ -32,7 +32,7 @@ QuicLogEmitter::~QuicLogEmitter() {
   }
   std::string content = stream_.str();
   if (!content.empty() && content.back() == '\n') {
-    // strip the last trailing '\n' because spd log will add a trailing '\n' to
+    // strip the last trailing '\n' because `spd` log will add a trailing '\n' to
```

Comment:
change to `spdlog`; it shouldn't be two words

Reply:
Done.